src/platform/logging/BUILD.gn

```diff
@@ -4,8 +4,90 @@ import("//build_overrides/build.gni")
 import("${chip_root}/src/platform/device.gni")
 
+static_library("default") {
+  if (!chip_use_external_logging) {
+    deps = []
+
+    if (chip_use_pw_logging) {
+      deps += [ "$dir_pw_log" ]
+    }
+
+    if (chip_device_platform == "cyw30739") {
+      deps += [ "${chip_root}/src/platform/Infineon/CYW30739:logging" ]
+    } else if (chip_device_platform == "psoc6") {
+      deps += [ "${chip_root}/src/platform/Infineon/PSOC6:logging" ]
+    } else if (chip_device_platform == "cc32xx") {
+      deps += [ "${chip_root}/src/platform/cc32xx:logging" ]
+    } else if (chip_device_platform == "esp32") {
+      deps += [ "${chip_root}/src/platform/ESP32:logging" ]
+    } else if (chip_device_platform == "openiotsdk") {
+      deps += [ "${chip_root}/src/platform/openiotsdk:logging" ]
+    } else if (chip_device_platform == "mbed") {
+      deps += [ "${chip_root}/src/platform/mbed:logging" ]
+    } else if (chip_device_platform == "bl602" ||
+               chip_device_platform == "bl702" ||
+               chip_device_platform == "bl702l") {
+      deps += [ "${chip_root}/src/platform/bouffalolab/common:logging" ]
+    } else if (chip_device_platform == "mt793x") {
+      deps += [ "${chip_root}/src/platform/mt793x:logging" ]
+    } else if (chip_device_platform == "cc13x2_26x2") {
+      deps += [ "${chip_root}/src/platform/cc13xx_26xx/cc13x4_26x2:logging" ]
+    } else if (chip_device_platform == "cc13x4_26x4") {
+      deps += [ "${chip_root}/src/platform/cc13xx_26xx/cc13x4_26x4:logging" ]
+    } else if (chip_device_platform == "ameba") {
+      deps += [ "${chip_root}/src/platform/Ameba:logging" ]
+    } else if (chip_device_platform == "asr") {
+      deps += [ "${chip_root}/src/platform/ASR:logging" ]
+    } else if (chip_device_platform == "webos") {
+      deps += [ "${chip_root}/src/platform/webos:logging" ]
+    } else if (chip_device_platform == "zephyr" ||
+               chip_device_platform == "nxp_zephyr") {
+      deps += [ "${chip_root}/src/platform/Zephyr:logging" ]
+    } else if (chip_device_platform == "nrfconnect") {
+      deps += [ "${chip_root}/src/platform/nrfconnect:logging" ]
+    } else if (chip_device_platform == "telink") {
+      deps += [ "${chip_root}/src/platform/telink:logging" ]
+    } else if (chip_device_platform == "efr32") {
+      deps += [ "${chip_root}/src/platform/silabs/efr32:logging" ]
+    } else if (chip_device_platform == "SiWx917") {
+      deps += [ "${chip_root}/src/platform/silabs/SiWx917:logging" ]
+    } else if (chip_device_platform == "linux") {
+      deps += [ "${chip_root}/src/platform/Linux:logging" ]
+    } else if (chip_device_platform == "tizen") {
+      deps += [ "${chip_root}/src/platform/Tizen:logging" ]
+    } else if (chip_device_platform == "qpg") {
+      deps += [ "${chip_root}/src/platform/qpg:logging" ]
+    } else if (chip_device_platform == "darwin") {
+      sources = [ "${chip_root}/src/platform/Darwin/LoggingImpl.cpp" ]
```

Comment:
Why this source is here and not like in other platforms within the logging target?

Reply:
Fixed